package.json

```diff
@@ -22,7 +22,7 @@
     "@types/mongodb": "^3.5.27",
     "bson": "^1.1.4",
     "kareem": "2.3.2",
-    "mongodb": "3.6.8",
+    "mongodb": "^3.6.8",
```

Comment:
Please fix this, this change isn't intentional. Make sure you do `git diff` before committing :)